glib/src/param_spec.rs

```diff
@@ -43,7 +43,7 @@ impl crate::value::ValueType for ParamSpec {
 unsafe impl<'a> crate::value::FromValue<'a> for ParamSpec {
     type Checker = crate::value::GenericValueTypeOrNoneChecker<Self>;
 
-    unsafe fn from_value(value: &'a crate::Value) -> Self {
+    unsafe fn from_value(value: &crate::Value) -> Self {
```

Comment:
Why this change, and why does this even compile? :)

Reply:
Heh, not an intended change.